app/models/customer.rb

```diff
@@ -81,7 +73,7 @@ def applicable_invoice_grace_period
   end
 
   def editable?
-    !attached_to_subscriptions? &&
+    subscriptions.none? &&
       applied_add_ons.none? &&
       applied_coupons.where.not(amount_currency: nil).none? &&
```

Comment:
Do you know why we have `where.not(amount_currency: nil)` here? 🤔

Reply:
@rsempe if we assign any resource holding currency to a customer, it becomes impossible to update the currency of the customer.
Regarding coupon, the limitation does not apply to "percentage" coupons as they do not hold a currency